packages/pwa-kit-extension-sdk/src/express/middleware/apply-application-extensions.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+// Third-Party Imports
+import {Request, Response, NextFunction, Application} from 'express'
+
+// Local Imports
+import {getApplicationExtensions} from '../placeholders/application-extensions'
+
+// Define the middleware function that modifies the app
+const applyApplicationExtensions = (app: Application) => {
+    const applicationExtensions = getApplicationExtensions()
+    applicationExtensions
+        .filter((applicationExtension: any) => applicationExtension.isEnabled())
```

Comment:
Should we use the types already defined instead of using `any`?

https://github.com/SalesforceCommerceCloud/pwa-kit/blob/74d7f24408c8eb2ba4760c3fdcc1bcc10b5a71fb/packages/pwa-kit-extension-sdk/src/express/placeholders/application-extensions.ts#L15